operator/src/main/resources/application-rhbk.properties

```diff
@@ -0,0 +1,7 @@
+operator.keycloak.pod-labels."com.company"=Red_Hat
+operator.keycloak.pod-labels."rht.prod_name"=Red_Hat_Runtimes
+operator.keycloak.pod-labels."rht.prod_ver"=${SSO_QUARTER:sso_quarter_placeholder}
+operator.keycloak.pod-labels."rht.comp"=SSO
+operator.keycloak.pod-labels."rht.comp_ver"=${SSO_VER:${quarkus.application.version}}
+operator.keycloak.pod-labels."rht.subcomp"=SSO_Server
+operator.keycloak.pod-labels."rht.subcomp_t"=application
```

Comment:
@ASzc Could you please check it's ok for the prod environment? `SSO_QUARTER` and `SSO_VER` are env vars with some default values (`quarkus.application.version` probably is reasonable default and might not even need overriding).

Please also note you need to build the Operator with the `-Poperator-prod` profile for this file to be used.

Reply:
This is implemented differently to how the golang one did it, but I assume you've checked and the labels are appropriately applied at runtime.

Example values:

```
SSO_QUARTER: 2023_Q2
SSO_VER: 7.6.3
```

So those defaults seem reasonable. I will probably set them anyway in the CSV

And noted. I can swap the profile after this is merged